modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md

```diff
@@ -158,8 +158,11 @@ April '18 - version 0.7
 
 ### Arrow 101 — Building an Android app using Functional Programming
 
-[Leandro Ferreira](https://twitter.com/mLeandroBF) presents how to build a simple app using Arrow, step by step. 
+[Leandro Ferreira](https://twitter.com/mLeandroBF) presents how to build a simple app using Arrow and how to implement
+solutions with Semigroups.
```

Comment:
There's an unnecessary line jump here. Ready to merge otherwise!

Reply:
Which line? 🤔

Reply:
Between implement and solutions. See the line indicator jumps from 161 to 162, even if the render looks okay.